webserver/src/main/java/io/kestra/webserver/controllers/api/ExecutionController.java

```diff
@@ -1735,14 +1734,15 @@ public HttpResponse<?> setLabelsByQuery(
     @Post(uri = "/{executionId}/unqueue")
     @Operation(tags = {"Executions"}, summary = "Unqueue an execution")
     public Execution unqueue(
-        @Parameter(description = "The execution id") @PathVariable String executionId
+        @Parameter(description = "The execution id") @PathVariable String executionId,
+        @Parameter(description = "The new state of the execution") @Nullable @QueryValue State.Type status
```

Comment:
```suggestion
        @Parameter(description = "The new state of the execution") @Nullable @QueryValue State.Type state
```

Please use `state` anywhere and not `status`